core/src/main/java/com/google/errorprone/bugpatterns/PreconditionsCheckNotNullPrimitive.java

```diff
@@ -157,10 +157,10 @@ private Fix createCheckArgumentOrStateCall(
 
     // Was the original call to Preconditions.checkNotNull a static import or not?
     if (methodInvocationTree.getMethodSelect().getKind() == Kind.IDENTIFIER) {
-      replacement.append(replacementMethod + "(");
+      replacement.append(replacementMethod).append("(");
```

Comment:
What about pulling this out of the if/else entirely? Then the else has append("Preconditions") and below has .append(replacementMethod).append('(') for both cases?